pkg/sql/delegate/show_grants.go

```diff
@@ -100,6 +102,33 @@ SELECT *
                ) AS a
        )
 `
+	//
+	const udfQuery = `
+WITH fn_grants AS (
+  SELECT routine_catalog as database_name,
+         routine_schema as schema_name,
+         reverse(split_part(reverse(specific_name), '_', 1))::OID as function_id,
+         routine_name as function_name,
+         grantee,
+         privilege_type,
+         is_grantable::boolean
+  FROM "".information_schema.role_routine_grants
+  WHERE reverse(split_part(reverse(specific_name), '_', 1))::INT > 100000
```

Comment:
I'm not including any builtin functions cause thinking it would just explode the `SHOW GRANTS` output without adding much user experience value since anyone can execute a builtin.